spec/lib/miq_automation_engine/models/miq_ae_yaml_import_export_spec.rb

```diff
@@ -197,8 +195,8 @@ def assert_import_failure_with_missing_file(import_options)
       end
 
       def assert_all_domains_imported(export_options, import_options)
-        export_model(ALL_DOMAINS, export_options)
-        reset_and_import(@export_dir, ALL_DOMAINS, import_options)
+        export_model(MiqAeYamlImportExportMixin::ALL_DOMAINS, export_options)
```

Comment:
I believe placing within a describe block puts it in whatever magical class RSpec defines to run the tests. IE (I'm making this up, don't quote me) `RSpec::TestCase::MiqAeDatastore` or something. 

> cue @jrafanie saying "RSpec's DSL is confusing...why can't we just use Ruby" ;)

I hear you :(